go.mod

```diff
@@ -106,4 +106,6 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/mfrancisc/toolchain-common v0.0.0-20221120225717-d72cdcb11147
```

Comment:
friendly reminder :-)

Reply:
fixed in [574bb7da30a68](https://github.com/codeready-toolchain/host-operator/commit/574bb7da30a684bf1c407130cc2d1f0b9315c5fb)